jetstream/__init__.py

```diff
@@ -5,11 +5,17 @@
 class AnalysisPeriod(enum.Enum):
     DAY = "day"
     WEEK = "week"
+    DAYS_28 = "days28"
     OVERALL = "overall"
 
     @property
-    def adjective(self) -> str:
-        d = {"day": "daily", "week": "weekly", "overall": "overall"}
+    def mozanalysis_label(self) -> str:
+        d = {"day": "daily", "week": "weekly", "days28": "28_day", "overall": "overall"}
+        return d[self.value]
+
+    @property
+    def table_suffix(self) -> str:
+        d = {"day": "daily", "week": "weekly", "days28": "days28", "overall": "overall"}
```

Comment:
I agree that `mozanalysis_label` and `table_suffix` are the most convenient way to work around the naming issues here